modules/primer-css/package.json

```diff
@@ -34,6 +34,7 @@
     "primer-buttons": "2.3.0",
     "primer-cards": "0.4.0",
     "primer-core": "6.3.0",
+    "primer-dropdown": "1.0.0",
```

Comment:
Should we start at `1.0.0`?

Reply:
Only reason I ask is we have some less than `1.0` I think we should only bump to `1` when we add the JS